atala-prism-sdk/src/commonMain/kotlin/io/iohk/atala/prism/walletsdk/prismagent/PrismAgent.kt

```diff
@@ -715,8 +739,14 @@ class PrismAgent {
                     )
                 pluto.storeCredential(storableCredential)
                 return credential
-            } ?: throw io.iohk.atala.prism.walletsdk.domain.models.UnknownError.SomethingWentWrongError("Thid should not be null")
-        } ?: throw io.iohk.atala.prism.walletsdk.domain.models.UnknownError.SomethingWentWrongError("Cannot find attachment base64 in message")
+            }
+                ?: throw io.iohk.atala.prism.walletsdk.domain.models.UnknownError.SomethingWentWrongError(
+                    "Thid should not be null"
+                )
+        }
+            ?: throw io.iohk.atala.prism.walletsdk.domain.models.UnknownError.SomethingWentWrongError(
+                "Cannot find attachment base64 in message"
```

Comment:
We need to create errors for each case of this, and not just use the something went wrong.